source/PreferencesPanel.h

```diff
@@ -63,6 +63,8 @@ class PreferencesPanel : public Panel {
 	char page = 'c';
 	std::string hoverPreference;
 
+	int settingsPage = 0;
```

Comment:
Euh whats settingsPage and pageCount? I'm kinda confused caus pageCount said how many pages of settings there are..?
Is this the current page being shown? And why would it start at 0 and not 1 then? cpp logic? x)

Reply:
`pageCount` is how many pages there are, and it actually ought to be `SETTINGS_PAGE_COUNT`.
`settingsPage` is what page we are displaying. Starts at 0 because it is essentially an index.

Reply:
okay but add a comment to it or make it currentSettingsPage and totalSettingPages

Reply:
I am a great fan of variable names like settingPageIndex